portal-backend/depmap/data_access/interface.py

```diff
@@ -308,6 +299,26 @@ def get_slice_data(slice_query: SliceQuery) -> pd.Series:
 
     else:
         raise Exception("Unrecognized slice query identifier type")
+    
+###############################################################
+# METHODS BELOW ARE SPECIAL WORKAROUNDS FOR COMPOUND DATASETS #
+###############################################################
+# In the future, all drug screen datasets will be indexed by compound instead of compound experiment.
+# These methods exist to ensure that both the legacy backend and breadbox are returning the same 
```

Comment:
I'd say `same shaped data`. There's some arbitrariness in the choice of which compound experiment gets picked in this transitionary period.  That being said, I appreciate the comments here capturing the motivation!